dsf-gdb/org.eclipse.cdt.dsf.gdb/src/org/eclipse/cdt/dsf/gdb/service/GDBMemory.java

```diff
@@ -314,7 +316,13 @@ public void eventDispatched(IExitedDMEvent event) {
 	@Override
 	public int getAddressSize(IMemoryDMContext context) {
 		Integer addressSize = fAddressSizes.get(context);
-		return (addressSize != null) ? addressSize.intValue() : 8;
+		assert addressSize != null;
```

Comment:
I'm not sure regarding `assert` here, since further code handles the case under assertion as expected situation